src/review/views.py

```diff
@@ -2223,15 +2224,15 @@ def draft_decision_text(request, article_id):
     if not decision:
         raise Http404
 
-    if decision in ['accept', 'reject']:
+    if decision in {ED.ACCEPT, ED.DECLINE}:
         decision_text = logic.get_decision_content(
             request=request,
             article=article,
             decision=decision,
             author_review_url=author_review_url,
         )
 
-    elif decision in ['minor_revisions', 'major_revisions']:
+    elif decision in {ED.MINOR_REVISIONS, ED.MAJOR_REVISIONS}:
         revision = models.RevisionRequest(
             article=article,
             editor=request.user,
```

Comment:
There's a type issue that is causing a mismatch in these if statements.

I manually tested this view and was getting a different type for `decision` than for `ED.ACCEPT` and the others:

```py
print(decision)
# EditorialDecisions.ACCEPT
print(type(decision))
# <class 'str'>
print(ED.ACCEPT)
# EditorialDecisions.ACCEPT
print(type(ED.ACCEPT))
# <enum 'EditorialDecisions'>
```

Maybe `enum` does not evaluate as a string on comparison?

Reply:
yep, Jenkins picked up the same problem